lib/globalize.js

```diff
@@ -0,0 +1,5 @@
+var path = require('path');
+var SG = require('strong-globalize');
+
+SG.SetRootDir(path.join(__dirname, '..'), { autonomousMsgLoading: 'all' });
+module.exports = SG();
```

Comment:
No EOL at EOF, please fix. I thought `eslint` was configured to catch this, could you PTAL?